apps/pie-docs/src/components/modal/overview/overview.md

```diff
@@ -0,0 +1,421 @@
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/modifier-footer-passive.png",
+    alt: "A modal with a pair of primary and ghost buttons in its footer.",
+    width: 600
+} %}
+
+---
+
+#### Sizes
+
+There are three responsive Modal sizes: large, medium and small. Choose a size that works best for the amount of Modal content you have.
+
+Modals with short messages should use the small Modal to avoid long single lines; for complex or larger content, medium or large Modal will be more suitable.
+
+{% componentDetailsTable {
+  tableData: sizes
+} %}
+
+### Large
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/size-large.png",
+    alt: "Large size modal.",
+    width: 968
+} %}
+
+### Medium
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/size-medium.png",
+    alt: "Medium size modal.",
+    width: 520
+} %}
+
+### Small
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/size-small.png",
+    alt: "Small size modal.",
+    width: 336
+} %}
+
+---
+
+## Narrow
+
+### Sizes
+
+#### Full screen
+
+Large modals will expand to fill the entire page width at narrow viewports.
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/narrow-size-full-screen.png",
+    alt: "A full screen modal on a mobile screen.",
+    width: 302
+} %}
+
+#### 75% width
+
+Medium and small modals will expand to 75% page width at narrow viewports.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/narrow-size-75-percent.png",
+    alt: "A modal with 75% width on a mobile screen.",
+    width: 336
+} %}
+
+### Modifiers
+
+#### Footer
+
+As well as the standard Modal footer modifiers, at a narrow size you have the option to have full width Buttons that are stacked.
+
+{% contentLayout %}
+  {% contentPageImage {
+      src: "../../../assets/img/components/modal/overview/narrow-modifier-footer-single-action.png",
+      alt: "A modal with a single primary button in its footer on a mobile screen.",
+      width: 336
+  } %}
+  {% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/narrow-modifier-footer-dual-action.png",
+    alt: "A modal with a single primary button in its footer on a mobile screen.",
+    width: 288
+  } %}
+{% endcontentLayout %}
+
+---
+
+## Behaviours
+
+### Loading
+
+In order to keep the size consistent when the content inside the Modal is loading, we have set a fixed height of 360px while the loading spinner is visible.
+
+{% contentPageImage {
+  src: "../../../assets/img/components/modal/overview/behaviour-loading.png",
+  alt: "Loading behaviour of a modal.",
+  width: 600
+} %}
+
+---
+
+## Layout
+
+### Position
+
+{% notification {
+  type: "information",
+  message: "Only applicable to Modals that aren’t full screen."
+} %}
+
+#### Center
+
+The Modal is positioned at the center of the screen by default.
```

Comment:
The casing for Modal and Button seems a little inconsistent on this page. I personally prefer lowercase but please can you ensure it's the same everywhere whichever you choose?